src/controllers/channel.controller.js

```diff
@@ -93,4 +94,25 @@ exports.getChannels = async (req, res)=>{
     {
         return response_500(res, "Error fetching channels", err);
     }
+}
+
+exports.removeUserFromChannel = async (req,res)=>
+{
+    try {
+        const{user,channel} = req.body;
+
+        const filteredChannel = channel.members.filter(user=>user===user._id);
+        const filteredUser = User.channels.filter(({channel})=>channel===channel._id);
+
+        const updatedChannel = await Channel.findByIdAndUpdate(channel._id,{members:filteredChannel});
+        const updatedUser = await User.findByIdAndUpdate(user._id,{channels:filteredUser});
+
+        return response_200(res,"User Removed from Channel Successfully")
```

Comment:
Where do you remove the user. 
Also read issue description agin there are two users involved one user is adding other is being added